dojo/test/views.py

```diff
@@ -70,7 +70,8 @@ def view_test(request, tid):
         available_note_types = find_available_notetypes(notes)
     files = test.files.all()
     person = request.user.username
-    findings = Finding.objects.filter(test=test).order_by('numerical_severity')
+    findings = Finding.objects.filter(test_id=tid).order_by('numerical_severity')\
+        .defer('url', 'description', 'impact', 'steps_to_reproduce', 'severity_justification', 'references', 'payload', 'line', 'file_path', 'files')
```

Comment:
I don't think this really helps when showing a couple of findings. If in the future someone wants to show these fields it will result in extra queries.